### PR TITLE
[WIP]Add AtlasMap extension eclipse/che#13922

### PR DESCRIPTION
**Pull request by apupier**
creating for reference, unable to test locally for now :(

see https://github.com/eclipse/che/issues/13922 for the things to check before considering it for merge

### CODE REVIEW

**Comment by l0rd** [comment]
@apupier what's the plan with this PR?

**Comment by apupier** [comment]
> @apupier what's the plan with this PR?

the plan is to keep it opened to have it in sight and easier to test new version until the AtlasMap extension integration with Theia and Che are fixed.
Unless, you prefer to close it for now as I'm don't know when it will come on the top of the priorities.

**Comment by nickboldt** [comment]
Is this still a work in progress? It's been a few months ... :(

**Comment by apupier** [comment]
> Is this still a work in progress? It's been a few months

yes, even if sporadic, and new problems arise at each new steps :-(

File: v3/plugins/redhat/atlasmap-viewer/0.0.2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+description: Opens a browser for AtlasMap Data Transformation editor
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/jboss-fuse/vscode-atlasmap
+category: Other
+firstPublicationDate: "2019-07-19"
+spec:
+  containers:
+    - image: "eclipse/che-remote-plugin-runner-java8:7.0.0-next"
+      name: vscode-atlasmap
+      memoryLimit: "512Mi"
```
[the remark this answers is not in the document]
Comment (amisevsk) [reply v3/plugins/redhat/atlasmap-viewer/0.0.2/meta.yaml]: This also suppresses the "a new process is listening on port 8585" message.

File: v3/plugins/redhat/vscode-atlasmap/0.0.5/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    attributes:
+      protocol: http
+  containers:
+  - image: "quay.io/eclipse/che-sidecar-java:11-f76ca45"
+    name: vscode-atlasmap
+    memoryLimit: "512Mi"
+    ports:
+      - exposedPort: 8585
+  extensions:
+  - https://download.jboss.org/jbosstools/vscode/snapshots/vscode-atlasmap/vscode-atlasmap-0.0.5-126.vsix
```
Comment (apupier) [inline v3/plugins/redhat/vscode-atlasmap/0.0.5/meta.yaml]: 0.0.5 not yet released so pointing to a snapshot for work in progress state

**Comment by apupier** [comment]
still not working, provided details here https://github.com/eclipse/che/issues/13922#issuecomment-591853089

File: v3/plugins/redhat/atlasmap-viewer/0.0.2/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (apupier) [reply v3/plugins/redhat/atlasmap-viewer/0.0.2/meta.yaml]: I changed to the suggestion but I still have the "a process is listening on port 8585":

Screenshot from 2020-02-27 10-05-32

**Comment by dmytro-ndp** [comment]
check Happy path tests triggering

**Comment by ericwill** [comment]
This PR has been open for awhile -- let's close it and open a new one when something is ready for review.

**Pull request closed without merging**